dashboard/app/models/user.rb

```diff
@@ -2587,4 +2592,41 @@ def code_review_groups
     errors.add(:parent_email) unless parent_email.nil? ||
       Cdo::EmailValidator.email_address?(parent_email)
   end
+
+  US_STATE_DROPDOWN_OPTIONS = {
```

Comment:
Should this const be moved to shared_constants.rb or maybe the application_helper? It's kind of long, and I could see it being used in the future for other address-related things.

Reply:
I think this is specific to the `us_state` column in the User model. A more generic solution would probably want to support territories and other things as well.

Reply:
Makes sense. Do you know why we treat `full_address` as a string instead of doing a more complete address object, with properties like street address, country, state, zip, etc?

Reply:
nope :(